src/ert/_c_wrappers/analysis/configuration.py

```diff
@@ -19,7 +19,7 @@ class UpdateStep(BaseModel):
     row_scaling_parameters: List[RowScalingParameter] = []
 
     @root_validator(pre=True)
-    def transform_parameters(cls, values):
+    def transform_parameters(cls, values: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
I think the type is as correct as it can be now, but just leaving this for reference (guessing the dict that comes in always has this format): 
![Screenshot 2023-07-31 at 09 27 52](https://github.com/equinor/ert/assets/11595637/b2eaf9cc-cb95-453d-89a0-128854abead4)

Reply:
Could be defined using `TypedDict`:
```python
class ParameterValues(TypedDict):
   name: str,
   observations: List[str],
   parameters: List[str]
```

same for return type, could be TypedDict inheriting from this, with the added property "row_scaling_parameters"